externals/LICENSE.RmlUi

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Nitpick: Was trying to keep everything lower case (LICENSE.rmlui)

Reply:
So the main reason I used the capitals is because this has to match the directory name (casing included). 
I can rename the directory rmlui, although that does differ from upstream, I dont really care too much. Let me now your preference,